unit_tests/kernels/UnitTestKernelUtils.h

```diff
@@ -547,12 +547,32 @@ class MomentumABLKernelHex8Mesh : public MomentumKernelHex8Mesh
     const double bcVel[3] = {0.0, 0.0, 0.0};
     MomentumKernelHex8Mesh::fill_mesh_and_init_fields(doPerturb, generateSidesets);
     stk::mesh::field_fill_component(vel, *velocity_);
+    velocity_->modify_on_host();
+    velocity_->sync_to_device();
+
```

Comment:
Question regarding new NGP field behavior: does it automatically pull the field to device on first creation of NgpField? Or do we have to explicitly do this from now on?

Reply:
That behavior is unchanged.  The first time an NgpField is requested, it will be created on device and synced against the current stk Field data.

There is a behavior change from the old ngp::FieldManager to the new nalu-specific NgpFieldManager, however.  The old ngp::FieldManager would tear down the NgpFields when it was destructed, so that they would be re-created (and implicitly synced) when requested again.  The new NgpFieldManager lets the fields persist past its own lifetime.

Some of the unit tests were implicitly relying on multiple creations/destructions of the old ngp::FieldManager, which gave an implicit sync to the device before it was needed in the kernel being tested.  Now that the NgpFieldManager lets NgpFields persist on Device, we needed to explicitly sync a few things.